src/components/HomePage/HomePage.js

```diff
@@ -1,19 +1,23 @@
 import styled from 'styled-components/macro'
+import Discover from '../Discover/Discover'
 import Header from '../Header/Header'
 
-export default function HomePage() {
+export default function HomePage({ handleLikePlace, likedPlaces, AllSights }) {
```

Comment:
As I read AllSights in capital letter, I thought about a component.

Reply:
Yes you are right, already changed it!